roofit/roofit/src/RooBernstein.cxx

```diff
@@ -84,10 +84,20 @@ RooBernstein::RooBernstein(const RooBernstein& other, const char* name) :
 
 ////////////////////////////////////////////////////////////////////////////////
 
+void RooBernstein::selectNormalizationRange(const char* rangeName, Bool_t force)
```

Comment:
What does this function do? It seems to forever fix what the Bernstein is normalised to. However, in `getAnalyticalIntegral()`, one *cannot* select the range for analytic integration.
Since it returns `0` there, the Bernstein will just switch to numeric integration and ignore everything you do in this PR.
Is this intended?

Reply:
Hi Stephan,

    This was introduced so that the pdf uses the variable's full range even when considering the subrange this is a similar fix to RooChebychev. ([selectNormalizationRange](https://root.cern.ch/doc/master/classRooChebychev.html#a3278bec6891cc4d0afe54e16bde0955e)) . I have update the analytical integral to behave correctly for integration over ranges in a commit that follows.

Cheers,
Rahul

Reply:
Ok, so let's document that.